athenahealth/lab_results.go

```diff
@@ -0,0 +1,303 @@
+package athenahealth
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+type LabResult struct {
+	AttachmentExists         bool   `json:"attachmentexists"`
+	Description              string `json:"description"`
+	ExactDuplicateDocumentID int    `json:"exactduplicatedocumentid"`
+	FacilityID               int    `json:"facilityid"`
+	InternalNote             string `json:"internalnote"`
+	IsReviewedByProvider     string `json:"isreviewedbyprovider"`
+	LabResultDate            string `json:"labresultdate"`
+	LabResultDateTime        string `json:"labresultdatetime"`
+	LabResultID              int    `json:"labresultid"`
+	LabResultLOINC           string `json:"labresultloinc"`
+	LabResultNote            string `json:"labresultnote"`
+	OrderID                  int    `json:"orderid"`
+	PatientNote              string `json:"patientnote"`
+	Priority                 string `json:"priority"`
+	ProviderID               int    `json:"providerid"`
+	ResultStatus             string `json:"resultstatus"`
+}
+
+type ListLabResultsOptions struct {
+	StartDate           *time.Time `json:"startdate"`
+	LabResultStatus     *string    `json:"labresultstatus"`
+	ShowHidden          *bool      `json:"showhidden"`
+	ShowAbnormalDetails *bool      `json:"showabnormaldetails"`
+	EndDate             *time.Time `json:"enddate"`
+	HideDuplicate       *bool      `json:"hideduplicate"`
+
+	Pagination *PaginationOptions
+}
+
+type listLabResultsResponse struct {
+	LabResults []*LabResult `json:"results"`
+
+	PaginationResponse
+}
+
+type ListLabResultsResult struct {
```

Comment:
unfortunate stutter but understandable and would be the most reasonably expected name